.github/workflows/coverage.yml

```diff
@@ -37,15 +37,27 @@ jobs:
     env:
       DISPLAY: ${{ ':0' }}
       MUSIC_INSTALL_DIR: /opt/MUSIC
-      MUSIC_VERSION: 1.2.0
+      MUSIC_VERSION: 1.2.1
 
     steps:
 
       - name: Install apt packages
         run: |
-          sudo apt-get install xfonts-100dpi build-essential doxygen lcov libboost-all-dev libopenmpi-dev libmpich-dev libx11-dev libxcomposite-dev mpich openmpi-bin gpg ninja-build flex bison libfl-dev
+          sudo apt-get install xfonts-100dpi build-essential doxygen lcov libboost-all-dev libmpich-dev libx11-dev libxcomposite-dev mpich gpg ninja-build flex bison libfl-dev
         shell: bash
 
+      - name: Install a new version of OpenMPI
```

Comment:
Open MPI does not yet support MPI 4. There is no point on installing it from sources, that will not fix the underlying issue. The right fix is to fix MUSIC as per my advice elsewhere to support both MPI-3 (current Open MPI) and MPI-4 (current MPICH) implementations.

PS: For installing MPI on GitHub actions, look at https://github.com/mpi4py/setup-mpi/